pages/[id].js

```diff
@@ -0,0 +1,16 @@
+import Item from "@/component/Item";
+import { useRouter } from "next/router";
+
+export default function ItemDetailsPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  if (!id) {
+    return null;
+  }
+  return (
+    <>
+      <Item id={id} />
+    </>
```

Comment:
Since there is only one element, you can remove the fragment.
```suggestion
      <Item id={id} />
```